pandas/tests/series/test_analytics.py

```diff
@@ -1455,6 +1455,16 @@ def test_nsmallest_nlargest(self):
         expected = s.sort_values().head(3)
         assert_series_equal(result, expected)
 
+        # GH 15297
+        s = Series([1] * 5, index=[1, 2, 3, 4, 5])
+        expected = Series([1] * 3, index=[1, 2, 3])
```

Comment:
@jreback are you sure we need to add to the docstring, I think it's obvious enough here. ``keep=first`` in this example, and the series values are all duplicated, so you would expect it to return ``[1, 2, 3]`` as the index regardless of using 
``nsmallest`` or ``nlargest``